### ISSUE TITLE
What should the folder structure be for my own dataset for training and validation?

### ISSUE THREAD

**Issue by JibKh** [NVIDIA/flownet2-pytorch#229]
If I want to use some other dataset for training/validation, what does the folder structure need to be?
The current example is confusing as to how I can implement my own dataset. I downloaded the MpiSintel Dataset but couldn't translate it to what was given in the example.

Current Example:
`# Example on MPISintel Final and Clean, with L1Loss on FlowNet2 model
python main.py --batch_size 8 --model FlowNet2 --loss=L1Loss --optimizer=Adam --optimizer_lr=1e-4 \
--training_dataset MpiSintelFinal --training_dataset_root /path/to/mpi-sintel/final/dataset  \
--validation_dataset MpiSintelClean --validation_dataset_root /path/to/mpi-sintel/clean/dataset`
